ndscan/experiment/default_analysis.py

```diff
@@ -127,7 +170,7 @@ def required_axes(self) -> Set[ParamHandle]:
 
     def describe_online_analyses(
         self, context: AnnotationContext
-    ) -> Tuple[List[Dict[str, Any]], Dict[str, Dict[str, Any]]]:
+    ) -> Tuple[List[Dict[str, Any]], Dict[str, FitDescription]]:
```

Comment:
This probably shouldn't be a `FitDescription`, as there might be other online analysis types.

Reply:
ack. I wasn't sure quite how to name this class. Maybe better to have an `AnalysisSchema` (or `AnalysisDescription` or bikeshed any other name) base class to go alongside `Analysis`? With the idea being that the analysis `Analysis` object lives experiment-side, while `AnalysisSchema` is broadcast to the applets etc. Then we have a natural `FitAnalysisSchema` object here to go with the new `FitAnalysis` class (`OnlineFit` replacement).